GUI/MainRepo.cs

```diff
@@ -48,25 +54,8 @@ public void UpdateRepo()
 
         private void UpdateRepo(object sender, DoWorkEventArgs e)
         {
-            try
```

Comment:
@Postremus reports we don't catch 404s any more, and I'm guessing the removal of this try block may be why.